me.id.webverify/webverifylib/src/main/java/me/id/webverifylib/IDmeWebVerify.java

```diff
@@ -243,9 +254,19 @@ private void SendDataBack(String response)
     {
         activity.getIntent().putExtra(IDME_WEB_VERIFY_RESPONSE, response);
         activity.setResult(Activity.RESULT_OK, activity.getIntent());
-
         activity.finish();
     }
 
-
+    /**
+     * Sends the error back to the activity that called the WebView.
+     *
+     * @param error The service error form the Web Request.
+     */
+    private void SendErrorBack(String error)
+    {
+        Log.e("Web Request Error", error);
+        activity.getIntent().putExtra(IDME_WEB_VERIFY_RESPONSE, error);
+        activity.setResult(Activity.RESULT_CANCELED);
```

Comment:
is `RESULT_CANCELED` the appropriate code? It sounds to me that the user cancelled something when actually an exception has been catched